chef_master/source/install_supermarket.rst

```diff
@@ -124,7 +124,7 @@ All of the keys under ``node['supermarket_omnibus']`` are written out as ``/etc/
 
 .. code-block:: ruby
 
-   default['supermarket_omnibus']['chef_server_url'] = 'https://chefserver.mycompany.com'
+   default['supermarket_omnibus']['chef_server_url'] = 'https://chefserver.mycompany.com:443'
```

Comment:
Not sure why this port would need to be specified as that's the default for `https://`. Is it to trigger in the reader "OH, we run SSL on a different port than that!" thoughts?